compiler/bindings/python/iree/compiler/tools/tf.py

```diff
@@ -146,7 +146,7 @@ def compile_saved_model(saved_model_dir: str, **kwargs):
         if options.import_only:
             if options.output_file:
                 return None
-            with open(tf_iree_input, "r") as f:
+            with open(tf_iree_input, "rb") as f:
```

Comment:
Colab notebooks should also be updated to handle bytecode, such as here: https://github.com/openxla/iree/blob/fb38d2092cefa8aca53b1ed67c35829cb039b06b/samples/variables_and_state/variables_and_state.ipynb#L185-L198


This code will read a bytecode file and print it as text:

```python
from iree.compiler import ir

module = ir.Module.parse(module_mlir_bytecode, ir.Context())
module.dump()
```

Reply:
https://github.com/openxla/iree/pull/14743 also added a command to `ir_tool` that can do this.

Reply:
Updating notebooks in https://github.com/openxla/iree/pull/14901